front-end/src/components/loginform.js

```diff
@@ -1,20 +1,25 @@
 import React, { Component } from 'react'
```

Comment:
Since login is working with the axios call now, let's just leave it with axios and we'll integrate with Redux later. For now, the goal here is to have redux set up in the code base so that when we do start needing to use it, it's already set up.  So take this part of the commit (that changes loginform.js) completely out.